prdoc/pr_4564.prdoc

```diff
@@ -0,0 +1,23 @@
+title: "Make `on_unbalanceds` work with `fungibles` `imbalances`"
+
+doc:
+  - audience: Node Operator
+    description: |
+        The `fungibles` `imbalances` cannot be handled by the default implementation of `on_unbalanceds` from the `OnUnbalanced` trait. This is because the `fungibles` `imbalances` types do not implement the `Imbalance` trait (and cannot with its current semantics). The `on_unbalanceds` function requires only the `merge` function for the imbalance type. In this PR, we provide the `TryMerge` trait, which can be implemented by all imbalance types and make `OnUnbalanced` require it instead `Imbalance`.
+
+        This introduces a breaking change to the `OnUnbalanced` trait, but it only requires the removal of an unnecessary type parameter.
+
+crates:
+  - name: frame-support
+    bump: major
```

Comment:
Hm the CI suggest minor here, but a public trait was change, so should be major IMHO.